_maps/map_files/Theseus/TGS_Theseus.dmm

```diff
@@ -2031,14 +2031,6 @@
 	icon_state = "sterile_green_side"
 	},
 /area/mainship/medical/medical_science)
-"afo" = (
-/obj/machinery/door_control/ai/interior{
```

Comment:
I assume these changes are unintended?

Reply:
Uh, I'm not sure actually. But that typepath is outdated, no?

Reply:
Only in your PR.
Could you do this on a different PR, like spooky suggested?

Reply:
How do I not include theseus in my pr